packages/dds/tree/src/treeFactory.ts

```diff
@@ -24,20 +24,7 @@ import {
 	TreeFieldFromImplicitField,
 	TreeView,
 } from "./class-tree";
-
-/**
- * Configuration to specialize a Tree DDS for a particular use.
- * @internal
- */
-export interface TreeOptions extends SharedTreeOptions {
-	/**
-	 * Name appended to {@link @fluidframework/datastore-definitions#IChannelFactory."type"} to identify this factory configuration.
-	 * @privateRemarks
-	 * TODO: evaluate if this design is a good idea, or if "subtype" should be removed.
-	 * TODO: evaluate if schematize should be separated from DDS construction.
```

Comment:
IIRC, Tony and Vlad have recommended we instead record creation options in the IChannelAttributes metadata.

...since I'm right here...